extension.json

```diff
@@ -32,6 +32,39 @@
 	"AutoloadClasses": {
 		"SemanticMediaWiki": "includes/SemanticMediaWiki.php"
 	},
+	"TestAutoloadNamespaces": {
+ 		"SMW\\Test\\": "tests/phpunit",
+ 		"SMW\\Tests\\": "tests/phpunit",
+ 		"SMW\\Tests\\Fixtures\\": "tests/phpunit/Fixtures",
+ 		"SMW\\Tests\\Utils\\": "tests/phpunit/Utils"
```

Comment:
These two seem superfluous to me if you already have `"SMW\\Tests\\": "tests/phpunit",`

Reply:
Thanks, done.